tools/openocd/openocd_repository.bzl

```diff
@@ -1,8 +1,8 @@
 def _get_platform_specific_config(os_name):
     _WINDOWS = {
-        "sha256": "1fb26bbcfd65dbabe747ce3c8467a1f1cece7253bde4a95de13c2267d422ed8b",
-        "prefix": "xpack-openocd-0.10.0-14",
-        "url": "https://github.com/xpack-dev-tools/openocd-xpack/releases/download/v0.10.0-14/xpack-openocd-0.10.0-14-win32-x64.zip",
+        "sha256": "b195e3c3e9fe3f4fe9dfd9780ff08336ef8ed3015a76cc883d9e21e57c456f33",
+        "prefix": "",
+        "url": "https://github.com/ntfreak/openocd/releases/download/v0.11.0/openocd-v0.11.0-i686-w64-mingw32.tar.gz",
```

Comment:
Can we leave this as using the xpack/eclipse distribution urls. Happy to update to v0.11.0 but I'd rather get the openocd binaries all from one source for consistency. The ntfreak distrobution only packages up the windows binaries for release whereas the xpack distribution packages for most OS's. See [here](https://github.com/xpack-dev-tools/openocd-xpack/releases/tag/v0.11.0-1) for urls and sha's.